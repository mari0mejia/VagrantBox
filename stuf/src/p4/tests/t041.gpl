string window_title = "Project 4 t041.gpl";

rectangle all_the_way_down[11];

//Wacky but it works
textbox wowsers
( x=50, y=50, text=
  all_the_way_down[all_the_way_down[all_the_way_down[0].w].w].w
  + " by " +
  all_the_way_down[all_the_way_down[all_the_way_down[0].w].w].w
);
