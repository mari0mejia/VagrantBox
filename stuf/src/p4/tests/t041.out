gpl.cpp::main()
  input file(tests/t041.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
rectangle all_the_way_down[0]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[1]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[2]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[3]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[4]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[5]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[6]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[7]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[8]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[9]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[10]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
string window_title = "Project 4 t041.gpl"
textbox wowsers
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string("10 by 10"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(50),
    y = int(50)
}
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().

User quit program. Printing the symbol table.
rectangle all_the_way_down[0]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[1]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[2]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[3]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[4]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[5]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[6]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[7]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[8]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[9]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
rectangle all_the_way_down[10]
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(0),
    y = int(0)
}
string window_title = "Project 4 t041.gpl"
textbox wowsers
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string("10 by 10"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(71),
    x = int(50),
    y = int(50)
}
